docs/internals/json-schema.rst

```diff
@@ -16,5 +16,7 @@ A schema for the current model can be generated with the appropriate Phalanx com
 
    * - Schema
      - Command
-   * - `secrets.yaml files </schemas/secrets.json>`__
+   * - `environment values files <../schemas/environment.json>`__
+     - :command:`phalanx environment schema`
+   * - `secrets.yaml files <../schemas/secrets.json>`__
```

Comment:
Doing external links to internal files is somewhat unsupported in Sphinx, which I think is part of the reason linkcheck is failing on these. The two options are:

1. Use the download role to indicate this is a download. I think linkcheck will handle this? e.g.

```rst
:download:`/schemas/secrets.json`
```

2. If you want the schema to be browseable, then create an rst page for the schema itself and use `literalinclude` to show the schema and include a separate download link to make it easy to download the file.

Reply:
I think download is exactly what I want; I just didn't know that existed. (I don't think the schemas themselves are all that readable, although they could be made better if I tweaked the metadata in the schema, which is on my long list of follow-up stuff.) Thank you! I'll give that a try.

Reply:
That still isn't *exactly* what I wanted since I have to refer to it by the `/extras/schemas/environment.json` path for the download role and it makes a second copy of the schema, but when combined with html_extra_path it functionally it does everything I want, so it's a definite improvement. Thanks!